web-user-guide/data-access-request.rst

```diff
@@ -0,0 +1,144 @@
+the *review* step or make the *approve*/*reject* steps final.
+
+The most simple workflow is:
+
+* no ``Under Review`` intermediate state
+* no ``Conditionally Approved`` intermediate state
+* ``Approved`` state is final
+* ``Rejected`` state is final
+
+.. image:: ../images/mica-dar-simple-flow.png
+
+When the ``Under Review`` intermediate state is activated (which is the default
+configuration), the flow is:
+
+.. image:: ../images/mica-dar-default-flow.png
+
+When the ``Conditionally Approved`` intermediate state is activated, the flow
+is:
+
+.. image:: ../images/mica-dar-conditionally-approved-flow.png
+
+When both ``Under Review`` + ``Conditionally Approved`` intermediate states are
+activated, the flow is:
+
+.. image:: ../images/mica-dar-under-review-conditionally-approved-flow.png
+
+The workflow is resumed in the following table:
+
+.. list-table::
+  :widths: 15 55 15 15
+  :header-rows: 1
+
+  * - Status
+    - Description
+    - From Status
+    - To Status
+  * - Opened
+    - The request is in an editable state.
+    -
+    - Submitted
+  * - Submitted
+    - | The request is submitted to the application officer and is not editable by the applicant.
+      |
+      | The application officer can (conditionally) approve or reject it or, if configured, go to an intermediate under review status.
+    - Opened
+    - | Under Review
+      | or
+
+      * Approved
+      * Rejected
+  * - Under Review
+    - | The request is being reviewed by the application officer. He can reopen, conditionally approve (if this state is activated), approve or reject the request.
+    - Submitted
+    - * Opened
+      * Approved
+      * Rejected
+      * Conditionally Approved
+      * Submitted
+  * - Approved
+    - | The request has been approved. If configured, the request can go back to the submitted or reviewed status.
+    - | Submitted
+      | or
+      | Under Review
+    - | if state is not final:
+      |
+      | Submitted
+      | or
+      | Under Review
+  * - Rejected
+    - | The request has been rejected. If configured, the request can go back to the submitted or reviewed status.
+    - | Submitted
+      | or
+      | Under Review
+    - | if state is not final:
+      |
+      | Submitted
+      | or
+      | Under Review
+  * - Conditionally Approved
+    - | The request was submitted, has been reviewed and some adjustments are required before a final approval.
+      |
+      | The applicant can edit its request and will re-submit it.
+    - | Submitted
+      | or
+      | Under Review
+    - Submitted
+
+Application Form
+~~~~~~~~~~~~~~~~
+
+The application form is configured by the administrator, who can also define the PDF template used to create a printable copy of the form (available by clicking on the *Download* button).
+
+Feasibility Inquiry
+~~~~~~~~~~~~~~~~~~~
+
+Researchers can fill in a short form to ensure the feasibility of a project. This can be a preliminary of the main application form or of an amendment.
```

Comment:
The description of Feasibility Inquiries seems to be a little different from the one above... (dar.rst Line 71)